example/jamfile.v2

```diff
@@ -12,6 +12,7 @@ project
     : requirements
 			<library>/boost/system//boost_system
 			<library>/boost/thread//boost_thread
+			<library>/boost//serialization
```

Comment:
This doesn't match the others - is it correct?

Reply:
you mean the library location? i don't know why it is different, but this is the correct one for serialization